smart-home-application/src/UniquePointer.cpp

```diff
@@ -0,0 +1,66 @@
+#include "UniquePointer.h"
```

Comment:
this file needs to be removed, cannot declare template classes methods in header and define them in source files